edge-modules/ModuleRestarter/src/Program.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace ModuleRestarter
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class Program
+    {
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("ModuleRestarter");
+        static readonly string ServiceClientConnectionString = Preconditions.CheckNonWhiteSpace(Settings.Current.ServiceClientConnectionString, "ServiceClientConnectionString");

```

Comment:
you don't need to define these static readonly variables as you can directly call though Settings.Current static variable.

Reply:
I thought this was a good pattern for two reasons:
- if you have to reuse the variable, you won't have to preface with Settings.Current each time
- provides a convenient place to perform preconditions.check (cleaner than in the main method)

I see Direct Method Cloud Sender does not use static readonly, so it might be a better idea to change my implementation for consistency. Do you think I should change for this reason?